ci/post-msg.py

```diff
@@ -36,17 +36,16 @@ def add_results_from_worker(file):
         results = json.load(f)
     if previous_results_from != results['results_from']:
         comment_body += "\n" + results['results_from']
-        previous_results_from = results['results_from'] 
+        previous_results_from = results['results_from']
     comment_body += "\n" + results['summary']
 
 with open(sys.argv[1], "r") as credsfile:
     creds = [x.strip() for x in credsfile.readlines()]
 
-if len(creds) != 2:
+if len(creds) != 3:
     print("ERROR: Incorrect number of lines in credentials file!")
```

Comment:
Should we sys_exit here as well? I notice that we do it in run_workload and prepare_worker.